src/project-tests/survey-form-tests.js

```diff
@@ -115,7 +115,16 @@ export default function createSurveyFormTests() {
             });
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
-                var dropdown = document.getElementById('dropdown');
+                var dropdownid = 'dropdown';
```

Comment:
@tbushman just to reduce some of this code a bit, I think we can just avoid defining this variable.